gamepanel/game.php

```diff
@@ -203,26 +269,37 @@ function titleBar()
 	{
 		$rightTop = '
 			<div class="titleBarRightSide">
-				<span class="gameTimeRemaining">'.$this->gameTimeRemaining().'</span>
-			</div>
-			';
+				<div>
+				<span class="gameTimeRemaining">'.$this->gameTimeRemaining().'</span></div>'.
+			'</div>';
 
-		$rightBottom = '<div class="titleBarRightSide">
-				<span class="gameHoursPerPhase">
-					'.$this->gameHoursPerPhase().'
-				</span>
-			</div>';
+		$rightMiddle = '<div class="titleBarRightSide">'.
+				'<div>'.
+					'<span class="gameHoursPerPhase">'.$this->gameHoursPerPhase().'</span>'.$this->phaseSwitchInfo().
+				'</div>';
+			
+
+				
+		$rightMiddle .= '</div>';
+		
+		$rightBottom = '<div class="titleBarRightSide">'.
+					l_t('%s excused missed turn','<span class="excusedNMRs">'.$this->excusedMissedTurns.'</span>
+					').
+				'</div>';
+		// $rightBottom .= $this->phaseSwitchInfo();

```

Comment:
I think this comment can be removed as well.

Reply:
Done